packages/dashboard/src/components/Card/CardFrame/CardFrame.tsx

```diff
@@ -74,7 +77,10 @@ export class CardFrame extends React.Component<ICardFrameProps, {}> {
               className={classNames.ellipsisButtonStyle}
               menuIconProps={{ iconName: 'More' }}
               split={false}
-              aria-label={'More'}
+              ariaLabel={cardFrameAccessibility ? cardFrameAccessibility!.ariaLabel : undefined}
```

Comment:
>ariaLabel [](start = 14, length = 9)

We should have default values if none are passed for label, description and few others. Same thing applies for other components as well.